desktop/libs/aws/src/aws/conf.py

```diff
@@ -108,24 +108,16 @@ def get_default_region():
   return get_region(conf=AWS_ACCOUNTS['default']) if 'default' in AWS_ACCOUNTS else get_region()
 
 
-def get_default_host():
-  '''Returns the S3 host when Raz is configured'''
-
-  if get_raz_api_url():
-    endpoint = get_raz_s3_default_bucket()
-    if endpoint:
-      return endpoint.get('host')
-
 def get_region(conf=None):
   global REGION_CACHED
 
   if REGION_CACHED is not None:
     return REGION_CACHED
   region = ''
 
-  if conf or get_default_host():
+  if conf:
     # First check the host/endpoint configuration
-    endpoint = get_default_host() or conf.HOST.get()
+    endpoint = conf.HOST.get()
```

Comment:
`conf.HOST` is still a static value, isn't it? So, a user can only browse buckets from only that region.

Reply:
Yes, the intention here is to not depend on core-site as that has a hardcoded value and has been flaky from the past.

If someone wants to browse bucket from different region, then they can atleast change the Hue config after this change. This is a shot term fix.

For long term, there is a task jira to dynamically get the region value based on the bucket user wants to access. At that time, we may even skip setting this config but still need to scope.